foodsaving/invitations/serializers.py

```diff
@@ -17,7 +17,9 @@ class Meta:
             UniqueTogetherValidator(
                 queryset=Invitation.objects.filter(expires_at__gte=timezone.now()),
                 fields=('email', 'group'),
-                message=_('An invitation has already been sent to this e-mail address')
+                message=_(
```

Comment:
I would keep this error message as is, it's getting shown when _creating_ a new invitation, not when resending it.

Reply:
Sure, you're right, I'll fix that!